web/client/codechecker_client/cmd_line_client.py

```diff
@@ -55,19 +55,26 @@ def init_logger(level, stream=None, logger_name='system'):
     LOG = logger.get_logger(logger_name)
 
 
-def filter_localdir_remote_run(
-        run_args: List[str]) -> Tuple[List[str], List[str]]:
-    """Filter out arguments which are local directory or remote run names."""
+def filter_local_file_remote_run(
+    run_args: List[str]
+) -> Tuple[List[str], List[str]]:
+    """
+    Filter out arguments which are local directory, baseline files or remote
+    run names.
+    """
     local_dirs = []
+    baseline_files = []
     run_names = []
 
     for r in run_args:
         if os.path.isdir(r):
             local_dirs.append(os.path.abspath(r))
+        elif os.path.isfile(r) and r.endswith(".baseline"):
```

Comment:
As I mentioned earlier, I think expecting `.baseline` extension is not necessary. It is enough to check if it is a file here.